README.md

```diff
@@ -19,19 +19,29 @@ StarRocks is a next-gen sub-second MPP database for full analysis scenarios, inc
 * Pre-aggregations, flat tables, star and snowflake schemas are supported and all run at enhanced speed.
 * StarRocks hybridizes serving and analytical processing(HSAP) in an easy way. The minimalist architectural design reduces the complexity and maintenance cost of StarRocks and increases its reliability and scalability. 
 
+## Upstream
+
+[Apache Doris](https://github.com/apache/incubator-doris) is the upstream of StarRocks. We are very grateful to Apache Doris community for contributing such an excellent OLAP database.
```

Comment:
Becuase Apache Doris has not graduated, it should be called Apache Doris(incubating)